Terminal.Gui/Drawing/ColorParseException.cs

```diff
@@ -0,0 +1,85 @@
+#nullable enable
+using System.Diagnostics.CodeAnalysis;
+namespace Terminal.Gui;
+
+/// <summary>
+///   An exception thrown when something goes wrong when trying to parse a <see cref="Color" />.
+/// </summary>
+/// <remarks>
+///   Contains additional information to help locate the problem.<br /> Not intended to be thrown by consumers.
+/// </remarks>
+public sealed class ColorParseException : FormatException {
```

Comment:
💕

Reply:
A note about this: This class is somewhat rudimentary and not currently used to the fullest extent possible, even by its current definition, but I wanted to define things that might be useful later on right away, especially if any issues come up down the road related to formatting/parsing. :)